fem/fe_coll.hpp

```diff
@@ -1305,6 +1305,46 @@ class Local_FECollection : public FiniteElementCollection
    virtual int GetContType() const { return DISCONTINUOUS; }
 };
 
-}
+/// Radial basis function collection
+class KernelFECollection : public FiniteElementCollection
```

Comment:
Can we add some reference or brief description what the finite element space is?

Reply:
Done. See https://github.com/mfem/mfem/blob/meshless-kernels/fem/fe_coll.hpp#L1308-L1312.